ui/src/app/views/workspace/component.ts

```diff
@@ -300,4 +341,18 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     return this.accessLevel === WorkspaceAccessLevel.OWNER;
   }
 
+  sendFilesToNotebookServer(notebooks): void {
```

Comment:
localizeNotebooks?

Also above, I think localize over send would be more readable.